get_dependencies.py

```diff
@@ -1,22 +1,21 @@
 import os
-import shutil
 import tarfile
 from zipfile import ZipFile
-import requests
+from utils import download
```

Comment:
Let's move the `download` function in get_dependencies.py, as it's the only place where we'll use it.